forms/serializers/form.py

```diff
@@ -301,7 +301,9 @@ def get_information_checks(self, obj):
                     {
                         "id": information_check.id,
                         "name": information_check.name,
-                        "preparer": information_check.preparer,
+                        "preparer": "{} {}".format(
+                            information_check.first_name, information_check.last_name
```

Comment:
I believe this should've actually been `information_check.preparer.first_name` and `.last_name` (now this gives `AttributeError: 'InformationCheck' object has no attribute 'first_name'`). However, what I meant was that this should work the same as e.g. the preparer field for a plot search, which returns something like:

```
    "preparer": {
        "id": 18,
        "username": "eleanor9eb725c1",
        "first_name": "Mervi",
        "last_name": "Annikki",
        "is_staff": false
    }
```
On a quick glance, that seems to be handled there with just a single line:
```py
class PlotSearchRetrieveSerializer(PlotSearchSerializerBase):
    preparer = UserSerializer()
    # ...
```